src/models/inventoryModels/inventoryModel.ts

```
@@ -1061,6 +1061,7 @@ inventorySchema.set("toJSON", {
     transform(_document, returnedObject) {
         delete returnedObject._id;
         delete returnedObject.__v;
+        delete returnedObject.accountOwnerId;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)